test/features/caching/email_cache_client_test.dart

```diff
@@ -1,3 +1,5 @@
+@TestOn('vm')
```

Comment:
what is it?

Reply:
Mark this file for testing only on Dart VM. More details here. [https://github.com/dart-lang/test/tree/master/pkgs/test#platform-selectors](https://github.com/dart-lang/test/tree/master/pkgs/test#platform-selectors)